plugins/ltac/g_obligations.mlg

```diff
@@ -12,6 +12,8 @@
   Syntax for the subtac terms and types.
   Elaborated from correctness/psyntax.ml4 by Jean-Christophe Filliâtre *)
 
+DECLARE PLUGIN "coq-core.plugins.ltac"
```

Comment:
Just a thought, but maybe we'd like to have `DECLARE PLUGIN "foo"` and `EXTENDS PLUGIN GRAMMAR "foo"` to be different things?

I say this because it seems to me that a plugin should call only once the `add_known_module` API in `Mltop`.

Reply:
Doesn't really seem like a useful distinction to me.